loading.bs

```diff
@@ -897,7 +962,8 @@ To <dfn lt="reading a body|read a body">read a body</dfn> from a [=read buffer=]
     : "digest/mi-sha256-03"
     ::
         1. If the length of |signature|'s [=exchange signature/integrity header=]
-            isn't 1, return a failure.
+            isn't 1, return an error string
```

Comment:
This needs rebasing (which will remove the line).

Reply:
Done